[2026-05-21] New failing test cases not related to PR changes

### 🐛 Describe the bug

New failing test cases in pull workflow not related to changes from the PR.
https://github.com/intel/torch-xpu-ops/actions/runs/26086070759/job/77151379761?pr=3293

Those test cases needs to be triaged if they are not false negatives due to PyTorch reverts or CI lack of synchronization.

Cases:
~~op_ut,third_party.torch-xpu-ops.test.xpu.dynamo.test_regional_inductor_xpu.RegionalInductorInvokeSubgraphTests,test_max_autotune_no_cudagraphs_serialize_False~~
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_control_flow_xpu.TestControlFlowTraced,test_while_loop_autograd_simple
~~op_ut,third_party.torch-xpu-ops.test.xpu.test_custom_ops_xpu.TestCustomOpAPI,test_no_grad_skips_autograd~~

<details>
<summary>Error log</summary>

```text
2026-05-24T22:45:25.3791974Z =================================== FAILURES ===================================
2026-05-24T22:45:25.3792331Z ____________ TestControlFlowTraced.test_while_loop_autograd_simple _____________
2026-05-24T22:45:25.3792718Z [gw7] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
2026-05-24T22:45:25.3793312Z Traceback (most recent call last):
2026-05-24T22:45:25.3804681Z   File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/functorch/test_control_flow_xpu.py", line 9034, in test_while_loop_autograd_simple
2026-05-24T22:45:25.3805250Z     self.assertExpectedInline(
2026-05-24T22:45:25.3805865Z   File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 3501, in assertExpectedInline
2026-05-24T22:45:25.3806512Z     return super().assertExpectedInline(actual if isinstance(actual, str) else str(actual), expect, skip + 1)
2026-05-24T22:45:25.3807108Z   File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 413, in assertExpectedInline
2026-05-24T22:45:25.3807535Z     assert_expected_inline(
2026-05-24T22:45:25.3807953Z   File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 378, in assert_expected_inline
2026-05-24T22:45:25.3808403Z     assert_eq(expect, actual, msg=help_text)
2026-05-24T22:45:25.3808887Z   File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 450, in assertMultiLineEqualMaybeCppStack
2026-05-24T22:45:25.3809375Z     self.assertMultiLineEqual(expect, actual, *args, **kwargs)
2026-05-24T22:45:25.3809811Z   File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 1226, in assertMultiLineEqual
2026-05-24T22:45:25.3810215Z     self.fail(self._formatMessage(msg, standardMsg))
2026-05-24T22:45:25.3810668Z   File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 675, in fail
2026-05-24T22:45:25.3811030Z     raise self.failureException(msg)
2026-05-24T22:45:25.3811533Z AssertionError: 'clas[152 chars]     zeros: "i64[]" = torch.ops.aten.zeros.def[3514 chars]1)\n' != 'clas[152 chars]     clone: "f32[3, 3]" = torch.ops.aten.clone[3636 chars]1)\n'
2026-05-24T22:45:25.3812038Z   class GraphModule(torch.nn.Module):
2026-05-24T22:45:25.3812413Z       def forward(self, primals_2: "f32[3, 3]", primals_3: "f32[3]", cat: "f32[u2, 3, 3]", tangents_1: "f32[3, 3]"):
2026-05-24T22:45:25.3812955Z -         zeros: "i64[]" = torch.ops.aten.zeros.default([], dtype = torch.int64, device = device(type='cpu'), pin_memory = False)
2026-05-24T22:45:25.3813537Z +         clone: "f32[3, 3]" = torch.ops.aten.clone.default(tangents_1, memory_format = torch.contiguous_format);  tangents_1 = None
2026-05-24T22:45:25.3814051Z           zeros_like: "f32[3]" = torch.ops.aten.zeros_like.default(primals_3, pin_memory = False)
2026-05-24T22:45:25.3814485Z           zeros_like_1: "f32[3, 3]" = torch.ops.aten.zeros_like.default(primals_2, pin_memory = False)
2026-05-24T22:45:25.3814857Z           while_loop_cond_graph_1 = self.while_loop_cond_graph_1
2026-05-24T22:45:25.3815159Z           while_loop_body_graph_1 = self.while_loop_body_graph_1
2026-05-24T22:45:25.3816026Z -         while_loop = torch.ops.higher_order.while_loop(while_loop_cond_graph_1, while_loop_body_graph_1, (zeros, tangents_1, zeros_like, zeros_like_1), (cat, primals_3, primals_2));  while_loop_cond_graph_1 = while_loop_body_graph_1 = zeros = tangents_1 = zeros_like = zeros_like_1 = cat = primals_3 = primals_2 = None
2026-05-24T22:45:25.3817015Z +         zeros: "i64[]" = torch.ops.aten.zeros.default([], dtype = torch.int64, device = device(type='cpu'), pin_memory = False)
2026-05-24T22:45:25.3817962Z +         while_loop = torch.ops.higher_order.while_loop(while_loop_cond_graph_1, while_loop_body_graph_1, (zeros, clone, zeros_like, zeros_like_1), (cat, primals_3, primals_2));  while_loop_cond_graph_1 = while_loop_body_graph_1 = zeros = clone = zeros_like = zeros_like_1 = cat = primals_3 = primals_2 = None
2026-05-24T22:45:25.3818748Z           getitem_2: "f32[3, 3]" = while_loop[1]
2026-05-24T22:45:25.3818999Z           getitem_3: "f32[3]" = while_loop[2]
2026-05-24T22:45:25.3819270Z           getitem_4: "f32[3, 3]" = while_loop[3];  while_loop = None
2026-05-24T22:45:25.3819543Z           return (getitem_2, getitem_4, getitem_3)
2026-05-24T22:45:25.3819869Z   
2026-05-24T22:45:25.3820076Z       class while_loop_cond_graph_1(torch.nn.Module):
2026-05-24T22:45:25.3820515Z           def forward(self, arg0_1: "i64[]", arg1_1: "f32[3, 3]", arg2_1: "f32[3]", arg3_1: "f32[3, 3]", arg4_1: "f32[u2, 3, 3]", arg5_1: "f32[3]", arg6_1: "f32[3, 3]"):
2026-05-24T22:45:25.3821039Z               sym_size_int_1: "Sym(u2)" = torch.ops.aten.sym_size.int(arg4_1, 0);  arg4_1 = None
2026-05-24T22:45:25.3821351Z   
2026-05-24T22:45:25.3821624Z               lt: "b8[]" = torch.ops.aten.lt.Scalar(arg0_1, sym_size_int_1);  arg0_1 = sym_size_int_1 = None
2026-05-24T22:45:25.3821963Z               return lt
2026-05-24T22:45:25.3822148Z   
2026-05-24T22:45:25.3822348Z       class while_loop_body_graph_1(torch.nn.Module):
2026-05-24T22:45:25.3822784Z           def forward(self, arg0_1: "i64[]", arg1_1: "f32[3, 3]", arg2_1: "f32[3]", arg3_1: "f32[3, 3]", arg4_1: "f32[u2, 3, 3]", arg5_1: "f32[3]", arg6_1: "f32[3, 3]"):
2026-05-24T22:45:25.3823246Z               sym_size_int_1: "Sym(u2)" = torch.ops.aten.sym_size.int(arg4_1, 0)
2026-05-24T22:45:25.3823509Z   
2026-05-24T22:45:25.3823784Z               rsub: "i64[]" = torch.ops.aten.rsub.Scalar(arg0_1, sym_size_int_1);  sym_size_int_1 = None
2026-05-24T22:45:25.3824176Z               sub_1: "i64[]" = torch.ops.aten.sub.Tensor(rsub, 1);  rsub = None
2026-05-24T22:45:25.3824581Z               _local_scalar_dense: "Sym(u7)" = torch.ops.aten._local_scalar_dense.default(sub_1);  sub_1 = None
2026-05-24T22:45:25.3825128Z               select: "f32[3, 3]" = torch.ops.aten.select.int(arg4_1, 0, _local_scalar_dense);  arg4_1 = _local_scalar_dense = None
2026-05-24T22:45:25.3825562Z               t: "f32[3, 3]" = torch.ops.aten.t.default(arg6_1);  arg6_1 = None
2026-05-24T22:45:25.3825885Z               t_1: "f32[3, 3]" = torch.ops.aten.t.default(t);  t = None
2026-05-24T22:45:25.3826196Z               mm: "f32[3, 3]" = torch.ops.aten.mm.default(arg1_1, t_1);  t_1 = None
2026-05-24T22:45:25.3826517Z               t_2: "f32[3, 3]" = torch.ops.aten.t.default(arg1_1)
2026-05-24T22:45:25.3826837Z               mm_1: "f32[3, 3]" = torch.ops.aten.mm.default(t_2, select);  t_2 = None
2026-05-24T22:45:25.3827172Z               t_3: "f32[3, 3]" = torch.ops.aten.t.default(mm_1);  mm_1 = None
2026-05-24T22:45:25.3827502Z               sum_1: "f32[1, 3]" = torch.ops.aten.sum.dim_IntList(arg1_1, [0], True)
2026-05-24T22:45:25.3827854Z               view: "f32[3]" = torch.ops.aten.view.default(sum_1, [3]);  sum_1 = None
2026-05-24T22:45:25.3828187Z               t_4: "f32[3, 3]" = torch.ops.aten.t.default(t_3);  t_3 = None
2026-05-24T22:45:25.3828486Z               mul_4: "f32[3, 3]" = torch.ops.aten.mul.Tensor(arg1_1, select)
2026-05-24T22:45:25.3828851Z               mul_5: "f32[3, 3]" = torch.ops.aten.mul.Tensor(arg1_1, select);  arg1_1 = select = None
2026-05-24T22:45:25.3829238Z               add_7: "f32[3, 3]" = torch.ops.aten.add.Tensor(mm, mul_5);  mm = mul_5 = None
2026-05-24T22:45:25.3829621Z               add_8: "f32[3, 3]" = torch.ops.aten.add.Tensor(add_7, mul_4);  add_7 = mul_4 = None
2026-05-24T22:45:25.3829990Z               add_9: "i64[]" = torch.ops.aten.add.Tensor(arg0_1, 1);  arg0_1 = None
2026-05-24T22:45:25.3830356Z               add_10: "f32[3]" = torch.ops.aten.add.Tensor(view, arg2_1);  view = arg2_1 = None
2026-05-24T22:45:25.3830741Z               add_11: "f32[3, 3]" = torch.ops.aten.add.Tensor(t_4, arg3_1);  t_4 = arg3_1 = None
2026-05-24T22:45:25.3831049Z               return (add_9, add_8, add_10, add_11)
2026-05-24T22:45:25.3831491Z  : To accept the new output, re-run test with envvar EXPECTTEST_ACCEPT=1 (we recommend staging/committing your changes before doing this)
2026-05-24T22:45:25.3831842Z 
2026-05-24T22:45:25.3831974Z To execute this test, run the following from the base repo dir:
2026-05-24T22:45:25.3832450Z     PYTORCH_TEST_WITH_SLOW=1 python test/xpu/functorch/test_control_flow_xpu.py TestControlFlowTraced.test_while_loop_autograd_simple
```

</details>

### Versions

Versions

--- a/test/xpu/functorch/test_control_flow_xpu.py
+++ b/test/xpu/functorch/test_control_flow_xpu.py
@@ -9036,12 +9036,13 @@ class GraphModule(torch.nn.Module):
                 """\
 class GraphModule(torch.nn.Module):
     def forward(self, primals_2: "f32[3, 3]", primals_3: "f32[3]", cat: "f32[u2, 3, 3]", tangents_1: "f32[3, 3]"):
-        zeros: "i64[]" = torch.ops.aten.zeros.default([], dtype = torch.int64, device = device(type='cpu'), pin_memory = False)
+        clone: "f32[3, 3]" = torch.ops.aten.clone.default(tangents_1, memory_format = torch.contiguous_format);  tangents_1 = None
         zeros_like: "f32[3]" = torch.ops.aten.zeros_like.default(primals_3, pin_memory = False)
         zeros_like_1: "f32[3, 3]" = torch.ops.aten.zeros_like.default(primals_2, pin_memory = False)
         while_loop_cond_graph_1 = self.while_loop_cond_graph_1
         while_loop_body_graph_1 = self.while_loop_body_graph_1
-        while_loop = torch.ops.higher_order.while_loop(while_loop_cond_graph_1, while_loop_body_graph_1, (zeros, tangents_1, zeros_like, zeros_like_1), (cat, primals_3, primals_2));  while_loop_cond_graph_1 = while_loop_body_graph_1 = zeros = tangents_1 = zeros_like = zeros_like_1 = cat = primals_3 = primals_2 = None
+        zeros: "i64[]" = torch.ops.aten.zeros.default([], dtype = torch.int64, device = device(type='cpu'), pin_memory = False)
+        while_loop = torch.ops.higher_order.while_loop(while_loop_cond_graph_1, while_loop_body_graph_1, (zeros, clone, zeros_like, zeros_like_1), (cat, primals_3, primals_2));  while_loop_cond_graph_1 = while_loop_body_graph_1 = zeros = clone = zeros_like = zeros_like_1 = cat = primals_3 = primals_2 = None
         getitem_2: "f32[3, 3]" = while_loop[1]
         getitem_3: "f32[3]" = while_loop[2]
         getitem_4: "f32[3, 3]" = while_loop[3];  while_loop = None

